pds_doi_core/actions/draft.py

```diff
@@ -83,30 +148,39 @@ def run(self,
             logger.info(f"permissible_contributor_list {o_permissible_contributor_list}")
             exit(1)
 
-        # parse input
-        if not input.startswith('http'):
-            xml_tree = etree.parse(input)
-        else:
-            response = requests.get(input)
-            xml_tree = etree.fromstring(response.content)
+        # The value of input can be a list of names, or a directory.  Resolve that to a list of names.
+        list_of_names = self._resolve_input_into_list_of_names(input)
+        # Return immediately if there are no files found.
+        # Note that exit() function is not used as it would interfere with the unit test.
+        if len(list_of_names) == 0:
```

Comment:
Yes, it can happen if the input for the -i is a directory and there is nothing inside.  As it turns out, we don't need to do the check since the while loop will take care of that (iterating over a list of zero). 

I removed len() check and the code still runs OK.  You are right.  There's no need to do an additional check when the for loop works just fine.